ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1485,6 +1646,57 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'UCU Heavensfall Nael Spot',
+      type: 'StartsUsing',
+      // Grab position data once Bahamut begins casting Megaflare Dive
+      netRegex: { id: '26E1', source: 'Bahamut Prime', capture: false },
+      condition: (data) => data.trio === 'heavensfall',
+      promise: async (data) => {
+        if (!data.trioSourceIds.nael || !data.trioSourceIds.twin || !data.trioSourceIds.bahamut)
+          return;
+        data.combatantData = [];
+        data.combatantData = (await callOverlayHandler({
+          call: 'getCombatants',
+          ids: [data.trioSourceIds.nael, data.trioSourceIds.bahamut, data.trioSourceIds.twin],
+        })).combatants;
+      },
+      alertText: (data, _matches, output) => {
+        // Bosses line up adjacent to one another, but don't necessarily have discrete directional positions (based on 8Dir scale).
+        // But we can calculate their position as an angle (relative to circular arena): 0 = N, 90 = E, 180 = S, 270 = W, etc.
+        // Set initial angles to -1 to avoid issues with nullish checks, as 0 is a valid angle and a nullish value.
```

Comment:
Could you set these to undefined and check against undefined explicitly on line 1682 rather than using a magic sentinel value here?